lib/cdo/delete_accounts_helper.rb

```diff
@@ -110,7 +110,6 @@ def clean_and_destroy_pd_content(user_id)
     Pd::FacilitatorProgramRegistration.where(user_id: user_id).update_all(form_data: '{}')
     Pd::RegionalPartnerProgramRegistration.where(user_id: user_id).update_all(form_data: '{}', teachercon: 0)
     Pd::Teachercon1819Registration.where(user_id: user_id).update_all(form_data: '{}', user_id: nil)
-    Pd::TeacherApplication.where(user_id: user_id).update_all(primary_email: '', secondary_email: '', application: '')
```

Comment:
@islemaster what's the best practice for deleting user data for models that no longer exist?

Reply:
If the database tables still exist, we can convert this to a raw SQL query.

Alternatively, if we're removing this model presumably our application isn't using this data anymore and we could archive it someplace else and remove the table entirely.  @breville did this recently - any recommendations?

Reply:
Yes, as part of https://github.com/code-dot-org/code-dot-org/pull/29987 we archived the data to a gsheet, which was reviewed by relevant people before we dropped the table.  That data was fairly independent; I can imagine it might be a bit more problematic for data that has a lot of references to other models, but it would be a case-by-case determination.